voltdbclient/url_test.go

```diff
@@ -0,0 +1,50 @@
+package voltdbclient
+
+import (
+	"testing"
+)
+
+func TestParseURL(t *testing.T) {
+	sample := []struct {
+		conn                       string
+		pass                       bool
+		scheme, user, passwd, host string
+	}{
+		{"https://localhost:21212", true, "https", "", "", "localhost:21212"},
+		{"http://localhost:21212", true, "http", "", "", "localhost:21212"},
+		{"localhost:21212", true, "localhost", "", "", "localhost:21212"},
+		{"127.0.0.1:21212", false, "voltdb", "", "", "localhost:21212"},
+		{"voltdb://", true, "voltdb", "", "", ":21212"},
+		{"voltdb://localhost", true, "voltdb", "", "", "localhost:21212"},
+		{"voltdb://localhost:21212", true, "voltdb", "", "", "localhost:21212"},
+		{"voltdb://localhost/mydb", true, "voltdb", "", "", "localhost:21212"},
```

Comment:
I don't think we support this now because `/mydb` would be meaningless to us at the moment.